src/Build.UnitTests/Utilities_Tests.cs

```diff
@@ -63,6 +60,23 @@ public void GetTextFromTextNodeWithXmlComment7()
             // Should get XML; note space after x added
             Assert.Equal("<!-- bar; baz; --><!-- bar --><x />", xmlContents);
         }
+
+        [Fact]
+        public void HandshakesDiffer()
+        {
+            int numHandshakeOptions = (int)Math.Pow(2, Enum.GetNames(HandshakeOptions.None.GetType()).Length - 1);
```

Comment:
You can do this at compile time instead

```suggestion
            int numHandshakeOptions = (int)Math.Pow(2, Enum.GetNames(typeof(HandshakeOptions)).Length - 1);
```